docs/components_page/components/input/colorpicker.py

```diff
@@ -0,0 +1,22 @@
+import dash_html_components as html
+from dash.dependencies import Input, Output
+import dash_bootstrap_components as dbc
+
+colorpicker = dbc.FormGroup(
+    [
+        dbc.Label("Choose the text color:", id="label"),
+        dbc.Input(
+            type="color",
+            id="colorpicker",
+            value="#000000",
+            style={"width": 75, "height": 50},
+        ),
+    ]
+)
+
+
+@app.callback(
+    Output("label", "style"), Input("colorpicker", "value"),
+)
+def update_text(color):
+    return {"color": color}
```

Comment:
This is cool! But I noticed that if you drag the selector around the color picker it fires the callback continuously, which might be a little bit of a problem if lots of readers of the documentation start doing it. Could we write this one as a clientside callback so we don't spam the server too much?

I think this problem can maybe be fixed in code, so that the callback is only triggered when you let go of the selector.

Reply:
Nice catch - I updated with a clienside callback, and I think it's a nice demo of this feature in Dash